app-misc/irtrans-irserver/irtrans-irserver-6.09.04-r1.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic toolchain-funcs mono-env multilib
+
+RESTRICT="strip"
+
+DESCRIPTION="Server software for IRTrans"
+HOMEPAGE="http://www.irtrans.de"
+SRC_URI="http://ftp.disconnected-by-peer.at/irtrans/irserver-src-${PV}.tar.gz
+	 http://ftp.disconnected-by-peer.at/irtrans/irserver-${PV}.tar.gz
+	http://www.irtrans.de/download/Server/Linux/irserver-src.tar.gz -> irserver-src-${PV}.tar.gz
+	http://www.irtrans.de/download/Server/Linux/irserver.tar.gz -> irserver-${PV}.tar.gz"
+
+LICENSE="BSD GPL-2"
+SLOT="0"
+KEYWORDS="~x86 ~amd64 ~arm"
```

Comment:
sort it

Reply:
LICENSE SLOT KEYWORDS IUSE is the stock ordering in the default ebuild when
using gentoo-syntax and is also the most common ordering for existing ebuilds.